src/ripple/consensus/LedgerTrie.h

```diff
@@ -575,14 +584,8 @@ class LedgerTrie
     std::uint32_t
     tipSupport(Ledger const& ledger) const
     {
-        Node const* loc;
-        Seq diffSeq;
-        std::tie(loc, diffSeq) = find(ledger);
-
-        // Exact match
-        if (loc && diffSeq == loc->span.end() && diffSeq > ledger.seq())
-            return loc->tipSupport;
-        return 0;
+        Node const* loc = findByLedgerID(ledger);
+        return loc ? loc->tipSupport : 0;
```

Comment:
This is more of a stylistic nit than anything, but what about:

    if (auto const* loc = findLedgerID(ledger))
        return loc->tipSupport;
    return 0;